README.rst

```diff
@@ -88,17 +88,29 @@ Installation
 
 #. Include these context processors::
 
+    Django 1.7::
     TEMPLATE_CONTEXT_PROCESSORS = (
         "django.core.context_processors.request",
         "treenav.context_processors.treenav_active",
     )
 
+    Django >= 1.8::
+    TEMPLATES = [
+      {
+        'OPTIONS': {
+          'context_processors': [
+            "django.template.context_processors.request",
+            "treenav.context_processors.treenav_active",
+          ],
+        },
+      },
+    ]
 
 #. Add these urls::
 
-    urlpatterns = patterns('',
+    urlpatterns = [
         (r'^treenav/', include('treenav.urls')),
```

Comment:
This tuple needs to be inside a `url` function since we're not using the `patterns` function anymore.
```
url(r'^treenav/', include('treenav.urls')),
```